src/com/twu/biblioteca/MainMenu.java

```diff
@@ -17,14 +17,24 @@ public void showList() {
         System.out.println("1. Clean Code | Robert C. Martin | 2008\n" + "2. Don't Make Me Think | Steve Krug | 2000\n" + "3. Test Driven Development | Kent Beck | 2000\n");
     }
 
+    public void showInvalidMessage() {
+        System.out.println("Please select a valid option!");
+    }
+
     public void manageOptions() {
-        // create scanner to read command-line input
+        int i = 0;
```

Comment:
There is a general convention in many languages that a variable named "i" is interpreted as an index into some collection of things, where the index is initialised to 0 at the start of a loop and then incremented through each pass of the loop.  Why am I saying this?  I believe you are using this variable for another purpose, in which case calling it "i" will confuse readers.